### PR TITLE
LPS-107835

### PR DESCRIPTION
**Pull request by yxingwu**

### CODE REVIEW

**Comment by yxingwu** [comment]
ci:test:sf

**Comment by yxingwu** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_kaleo_definition_version.jsp [hunk 1]
```diff
@@ -113,10 +113,10 @@ String successMessageKey = KaleoDesignerPortletKeys.KALEO_DESIGNER + "requestPro
 								<%= dateFormatTime.format(kaleoDefinitionVersion.getModifiedDate()) %>
 							</c:when>
 							<c:when test="<%= isPreviewBeforeRestoreState %>">
-								<liferay-ui:message arguments="<%= new String[] {dateFormatTime.format(kaleoDefinitionVersion.getModifiedDate()), userName} %>" key="revision-from-x-by-x" translateArguments="<%= false %>" />
+								<liferay-ui:message arguments="<%= new String[] {dateFormatTime.format(kaleoDefinitionVersion.getModifiedDate()), HtmlUtil.escape(userName)} %>" key="revision-from-x-by-x" translateArguments="<%= false %>" />
```
Comment (inacionery) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_kaleo_definition_version.jsp]: @yxingwu, isn't it better to escape directly to [kaleoDesignerDisplayContext](https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/display/context/KaleoDesignerDisplayContext.java#L539)?

Comment (topolik) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_kaleo_definition_version.jsp]: Hi @inacionery, 

this is safer. It's important to apply context-sensitive escaping. Inside KaleoDesignedDisplayContext it's not known what context escaping to apply. For it to be safe we would have to express the contract somewhere, for example in the method name ... `getUserNameEscapedForHTMLContext()`.

Comment (inacionery) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_kaleo_definition_version.jsp]: Alright!

**Comment by inacionery** [comment]
Hey @topolik,

When do you think you will complete this review?

Thanks!

**Comment by topolik** [comment]
Pull request submitted at: https://github.com/inacionery/liferay-portal/pull/582

View just my changes: https://github.com/topolik/liferay-portal/compare/yxingwu:3b67d62f8e...c4df23c90f
View total diff: https://github.com/topolik/liferay-portal/compare/60df34d7b2...c4df23c90f

**Pull request closed without merging**